### PR TITLE
Build on Mac

### PR DESCRIPTION
**Pull request by robinbb**

### CODE REVIEW

**Comment by robinbb** [comment]
Fixes #1.

**Comment by facundominguez** [comment]
@robinbb, how about CI is used to check the builds of these packages? Do we need to have reviewers building packages every time a change is contributed?

**Comment by robinbb** [comment]
@facundominguez Yes, CI should be used to check builds of these packages. I have made an issue for that. That task falls under the umbrella of #1 fortunately, so we have a mandate to do it. Until that is complete, we will have to do without CI testing builds.

File: jni/shell.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ nixpkgs ? import <nixpkgs> {}
+}:
+
+with nixpkgs;
```
Comment (facundominguez) [inline jni/shell.nix]: probably:
```
with nixpkgs // { inherit ghc };
```
plus adding `ghc` to the list of arguments of `shell.nix`.

**Comment by facundominguez** [comment]
`sparkle` is pinning the version of nixpkgs, should these packages do the same?

File: jni/shell.nix (hunk 1 above)
Comment (robinbb) [inline jni/shell.nix]: Good catch. I will correct this in another commit, soon.

**Comment by robinbb** [comment]
@facundominguez I have addressed your concerns. Please have another look.

**Comment by facundominguez** [comment]
`stack --nix install`  succeeds in both `jvm` and `jni`.

**Pull request merged**